### PR TITLE
feature/add-transcript-schema

### PR DESCRIPTION
**Pull request by tohuynh**
**Pull request recommendations:**
- [x] Name your pull request _your-development-type/short-description_. Ex: _feature/read-tiff-files_
- [ ] Link to any relevant issue in the PR description. Ex: _Resolves [gh-12], adds tiff file format support_
- [x] Provide context of changes.
I don't see that there should be any new addition to the current schema for `transcript`. So the design is exactly the same.
I added `transcript_file_id`, `transcript_file_uri` to `event` collection for faster querying of transcript content.
- [ ] Provide relevant tests for your feature or bug fix.
- [x] Provide or update documentation for any feature added by your pull request.

Thanks for contributing!

### CODE REVIEW

File: database_schema.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ event_id: {
     thumbnail_hover_uri: str
     video_uri: str
     transcript_id: str
+    transcript_file_id: str
+    transcript_file_uri: str
```
Comment (JacksonMaxfield) [inline database_schema.md]: I actually think we may want to take out all the transcript details from the event collection. Any ideas when we would need it for "fast ui access"?

File: database_schema.md [hunk 2]
```diff
@@ -153,6 +155,14 @@ A reference typing a specific person, and an event minutes item together.
...
 ### Transcript
 The primary transcript for an event.
+```
+transcript_id: {
+    event_id: str
+    file_id: str
+    confidence: float
+    created: datetime
+}
+```
```
Comment (JacksonMaxfield) [inline database_schema.md]: Same from last time, fine with me.

File: database_schema.md (hunk 1 above)
Comment (tohuynh) [reply database_schema.md]: For an event, don't we at least need the URI in order to download the  JSON to display transcript content?

Comment (JacksonMaxfield) [reply database_schema.md]: But that isn't needed for the cards. We have to be careful of _too_ much duplication of data. The event collection is meant for yes, event details, but also the _minimal amount of details needed to query and display event cards_ not the entire event route.

At least from my understanding

Comment (JacksonMaxfield) [reply database_schema.md]: On the event route, querying transcripts that match `event_id` and sorting by `confidence` is easier than continually updating the event collection every time we update a transcript version (if we ever do)

Comment (tohuynh) [reply database_schema.md]: I see. I thought `event` was supposed to contain everything needed for the event route.
I'm OK with `event` containing only the data needed for event cards. In that case, I agree with removing lines 20-22.

**Pull request merged**